src/components/AddMovieButton/AddMovieButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './AddMovieButton.module.scss';
+import AddMovieModal from '~/components/modals/AddMovieModal';
+
+class AddMovieButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { showAddMovieForm: false };
+
+    this.onAddMovieAction = this.onAddMovieAction.bind(this);
```

Comment:
it's better to use arrow function instead of binding

Reply:
Done. But had to add @babel/plugin-proposal-class-properties and appropriate linting